crud-routing/burger.router.js

```diff
@@ -0,0 +1,81 @@
+const burgerRouter = require('express').Router();
+const bodyParser = require('body-parser');
+const fs = require('fs');
+require('dotenv').config();
+const menuFile = process.env.MENU_FILE;
+
+burgerRouter.use(bodyParser.json());
+
+// get all burgers
+burgerRouter.get('/', (req, res) => {
+    const menuFile = process.env.MENU_FILE;
+    const data = JSON.parse(fs.readFileSync(menuFile));
+    res.json(data.burger);
+});
+
+// get burger by id
+burgerRouter.get('/:id', (req, res) => {
+    const menuFile = process.env.MENU_FILE;
+    const data = JSON.parse(fs.readFileSync(menuFile, 'utf8'));
```

Comment:
use async methods for all file operations

Reply:
used async method for all file operations